pySDC/projects/PinTSimE/battery_model.py

```diff
@@ -1,11 +1,11 @@
 import numpy as np
 import dill
 
-from pySDC.helpers.stats_helper import filter_stats, sort_stats
+from pySDC.helpers.stats_helper import sort_stats, filter_stats, get_sorted
 from pySDC.implementations.collocations import Collocation
```

Comment:
Please make consistent use of `get_sorted` and remove the two others from the import (and below)